Uplift/ProfileView.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  ProfileView.swift
+//  Uplift
+//
+//  Created by Cameron Hamidi on 3/1/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import SnapKit
+import UIKit
+
+class ProfileView: UIView {
+
+    // MARK: - Views
+    private let collectionView: UICollectionView!
+    private let gamesPlayedLabel = UILabel()
+    private let nameLabel = UILabel()
+    private let profilePicture = UIImageView(frame: .zero)
+
+    let collectionViewLeftRightInset: CGFloat = 24
+    let gamesSections: [GamesListHeaderSections] = [.myGames, .joinedGames, .pastGames]
+    let profilePictureSize: CGFloat = 60
+
+    // MARK: - Data
+    var myGames: [Post] = []
+    var joinedGames: [Post] = []
+    var pastGames: [Post] = []
+
+    override init(frame: CGRect) {
+        let layout = UICollectionViewFlowLayout()
+        layout.minimumLineSpacing = 12
+
+        collectionView = UICollectionView(frame: frame, collectionViewLayout: layout)
+
+        super.init(frame: frame)
```

Comment:
pretty sure we can just pass in `frame: .zero` to the collection view since we end up constraining it later on. this means that you can just have `super.init(frame: frame)` be the very first line of the function

Reply:
The `collectionView` initialization needs to come before the `super.init` since it's a let variable